fix/fixer_azure_exclude_from_latest.go

```diff
@@ -0,0 +1,58 @@
+package fix
+
+import (
+	"strings"
+
+	"github.com/mitchellh/mapstructure"
+)
+
+// FixerAzureExcludeFromLatest fix the spelling of "exclude_from_latest"
+// template in an Azure builder
+type FixerAzureExcludeFromLatest struct{}
+
+func (FixerAzureExcludeFromLatest) DeprecatedOptions() []string {
+	return []string{"exlude_from_latest"}
+}
+
+func (FixerAzureExcludeFromLatest) Fix(input map[string]interface{}) (map[string]interface{}, error) {
+	// The type we'll decode into; we only care about builders
+	type template struct {
+		Builders []map[string]interface{}
+	}
+
+	// Decode the input into our structure, if we can
+	var tpl template
+	if err := mapstructure.Decode(input, &tpl); err != nil {
+		return nil, err
+	}
+
+	for _, builder := range tpl.Builders {
+		builderTypeRaw, ok := builder["type"]
+		if !ok {
+			continue
+		}
+
+		builderType, ok := builderTypeRaw.(string)
+		if !ok {
+			continue
+		}
+
+		if !strings.HasPrefix(builderType, "azure-chroot") {
+			continue
+		}
+
+		excludeFromLatest, ok := builder["exlude_from_latest"]
```

Comment:
Hi there this is on the right track. The `"exclude_from_latest"` configuration option is actually nested with the `"shared_image_destination"` configuration block so this check needs to be dropped down a level. 

```go
   sharedImageDestination, ok := builder["shared_image_destination"].(map[string]interface{})
    if !ok {
      continue
    }

    excludeFromLatest, ok := sharedImageDestination["exlude_from_latest"]
    if !ok {
      continue
    }

    sharedImageDestination["exclude_from_latest"] = excludeFromLatest
    delete(sharedImageDestination, "exlude_from_latest")

    builder["shared_image_destination"] = sharedImageDestination
```

Once we the shared image destination block has been updated we can than assign it back to the builder.